evennia/contrib/mail.py

```diff
@@ -0,0 +1,220 @@
+"""
+In-Game Mail system
+
+Evennia Contribution - grungies1138 2016
+
+A simple Brandymail style @mail system that uses the Msg class from Evennia Core.
+
+Installation:
+    import MailCommand from this module into the default Player or Character command set
+"""
+
+from evennia import default_cmds, search_object
+from evennia.utils import create, utils, evtable, evform
+from evennia.comms.models import Msg
+
+
+_HEAD_CHAR = "|015-|n"
+_SUB_HEAD_CHAR = "-"
+_WIDTH = 78
+
+class CmdMail(default_cmds.MuxCommand):
+    """
+    Commands that allow either IC or OOC communications
+
+    Usage:    
+        {w@mail{n
```

Comment:
The `{` were not corrected, but looking at it, I think it's probably better to ditch the coloration in the help entry. It will not work well with the API docs.